lib/perl/Genome/Model/Tools/Vcflib.pm

```diff
@@ -0,0 +1,182 @@
+            'vcf2sqlite.py'              => '/usr/share/vcflib-tgi1.0/vcf2sqlite.py',
+            vcf_strip_extra_headers      => '/usr/share/vcflib-tgi1.0/vcf_strip_extra_headers',
+            vcfbiallelic                 => '/usr/share/vcflib-tgi1.0/vcfbiallelic',
+            vcfclearid                   => '/usr/share/vcflib-tgi1.0/vcfclearid',
+            vcfclearinfo                 => '/usr/share/vcflib-tgi1.0/vcfclearinfo',
+            vcfcomplex                   => '/usr/share/vcflib-tgi1.0/vcfcomplex',
+            vcffirstheader               => '/usr/share/vcflib-tgi1.0/vcffirstheader',
+            'vcfgtcompare.sh'            => '/usr/share/vcflib-tgi1.0/vcfgtcompare.sh',
+            vcfindelproximity            => '/usr/share/vcflib-tgi1.0/vcfindelproximity',
+            vcfindels                    => '/usr/share/vcflib-tgi1.0/vcfindels',
+            vcfmultiallelic              => '/usr/share/vcflib-tgi1.0/vcfmultiallelic',
+            vcfmultiway                  => '/usr/share/vcflib-tgi1.0/vcfmultiway',
+            vcfmultiwayscripts           => '/usr/share/vcflib-tgi1.0/vcfmultiwayscripts',
+            vcfnobiallelicsnps           => '/usr/share/vcflib-tgi1.0/vcfnobiallelicsnps',
+            vcfnoindels                  => '/usr/share/vcflib-tgi1.0/vcfnoindels',
+            vcfnosnps                    => '/usr/share/vcflib-tgi1.0/vcfnosnps',
+            vcfnulldotslashdot           => '/usr/share/vcflib-tgi1.0/vcfnulldotslashdot',
+            'vcfplotaltdiscrepancy.r'    => '/usr/share/vcflib-tgi1.0/vcfplotaltdiscrepancy.r',
+            'vcfplotaltdiscrepancy.sh'   => '/usr/share/vcflib-tgi1.0/vcfplotaltdiscrepancy.sh',
+            'vcfplotsitediscrepancy.r'   => '/usr/share/vcflib-tgi1.0/vcfplotsitediscrepancy.r',
+            'vcfplottstv.sh'             => '/usr/share/vcflib-tgi1.0/vcfplottstv.sh',
+            'vcfprintaltdiscrepancy.r'   => '/usr/share/vcflib-tgi1.0/vcfprintaltdiscrepancy.r',
+            'vcfprintaltdiscrepancy.sh'  => '/usr/share/vcflib-tgi1.0/vcfprintaltdiscrepancy.sh',
+            vcfqualfilter                => '/usr/share/vcflib-tgi1.0/vcfqualfilter',
+            vcfregionreduce              => '/usr/share/vcflib-tgi1.0/vcfregionreduce',
+            vcfregionreduce_and_cut      => '/usr/share/vcflib-tgi1.0/vcfregionreduce_and_cut',
+            vcfregionreduce_pipe         => '/usr/share/vcflib-tgi1.0/vcfregionreduce_pipe',
+            vcfregionreduce_uncompressed => '/usr/share/vcflib-tgi1.0/vcfregionreduce_uncompressed',
+            vcfremovenonATGC             => '/usr/share/vcflib-tgi1.0/vcfremovenonATGC',
+            vcfsnps                      => '/usr/share/vcflib-tgi1.0/vcfsnps',
+            vcfsort                      => '/usr/share/vcflib-tgi1.0/vcfsort',
+            vcfvarstats                  => '/usr/share/vcflib-tgi1.0/vcfvarstats',
+        }
+    );
+
+    if (exists $toolset{$version}) {
```

Comment:
It might be nice if the error handling in `sub vcflib_tool_path` and this were more similar.  That one dies if the conditional is true and returns otherwise.  This one returns if the conditional is true and dies otherwise.